### PR TITLE
feat : add navigation component

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
added navigation component in  component library.

### CODE REVIEW

File: Components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.navigation {
+    display: flex;
+    flex-direction: row;
+    background-color: var(--secondary-hover);
+    height: 2.5rem;
+    padding: 0.5rem 0.5rem;
```
Comment (adith-a-danthi) [inline Components/navigation.css]: You can just mention `padding: 0.5rem` since it is the same for all sides

**Review by adith-a-danthi — CHANGES_REQUESTED** [review]
Looks Good!

File: Components/navigation.css (hunk 1 above)
Comment (Omkar-Chaskar) [reply Components/navigation.css]: thanks. i will do it.

**Review by Radheshyam1234 — COMMENTED** [review]
looks good to merge

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me.

**Pull request merged**